darts/metrics/metrics.py

```diff
@@ -113,9 +114,28 @@ def wrapper_multivariate_support(*args, **kwargs):
     return wrapper_multivariate_support
 
 
+def _get_values(series: TimeSeries,
+                stochastic_quantile: float = 0.5) -> np.ndarray:
+    """
+    Returns the numpy values of a time series.
+    For stochastic series, return either all sample values with (stochastic_quantile=None) or the quantile sample value
+    with (stochastic_quantile {>=0,<=1})
+    """
+    if series.is_deterministic:
+        series_values = series.univariate_values()
+    else:  # stochastic
+        if stochastic_quantile is None:
+            series_values = series.data_array().values
+        else:
+            raise_if_not(isinstance(stochastic_quantile, float), 'stochastic quantile must be a float')
```

Comment:
Not sure we need this, since the function signature specifies that we need a (optional) float. Otherwise we would need checks for all function parameter data types in all other functions too, which is generally not what we do. @hrzn thoghts?

Reply:
I think this makes sense, thanks. Good to know for future PRs.